README.md

```diff
@@ -65,6 +65,7 @@ $ sed -i'' "s/namespace:.*/namespace: $NAMESPACE/g" ./deploy/rbac.yaml
 $ oc create -f deploy/rbac.yaml
 $ oc create role use-scc-hostmount-anyuid --verb=use --resource=scc --resource-name=hostmount-anyuid -n $NAMESPACE
 $ oc adm policy add-role-to-user use-scc-hostmount-anyuid system:serviceaccount:$NAMESPACE:nfs-client-provisioner
+$ oc adm policy add-scc-to-user hostmount-anyuid -z nfs-client-provisioner
```

Comment:
AFAIC, the SA name now is `$NAMESPACE-nfs-subdir-external-provisioner` (also affects the line above)